src/servicebus/azext_servicebus/custom.py

```diff
@@ -0,0 +1,174 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.command_modules.servicebus._utils import skunameconverter, skutireconverter, accessrights_converter
+# from azure.mgmt.servicebus.models import ErrorResponseException
+
+from azure.cli.core.commands.client_factory import get_subscription_id
+from azure.mgmt.servicebus.models import (
+    SBNamespace, SBSku, SBQueue, SBTopic, SBSubscription, Rule, Action, SqlFilter, CorrelationFilter, ArmDisasterRecovery)
+
+# from azure.mgmt.servicebus.models.service_bus_management_client_enums import SkuName, SkuTier, AccessRights, KeyType
+
+
+# Namespace Region
+def cli_namespace_create(client, resource_group_name, namespace_name, location, tags=None, sku='Standard', skutier=None,
+                         capacity=None):
+    result = client.create_or_update(resource_group_name, namespace_name, SBNamespace(location, tags,
+                                                                                      SBSku(skunameconverter(sku),
+                                                                                            skutireconverter(skutier),
+                                                                                            capacity=capacity)))
+    return result
+
+
+def cli_namespace_list(client, resource_group_name=None, namespace_name=None):
+    if resource_group_name and namespace_name:
+        result = client.get(resource_group_name, namespace_name)
+    return result
+
+    if resource_group_name and not namespace_name:
+        result = list(client.list_by_resource_group(resource_group_name, namespace_name))
+        return result
+
+    if not resource_group_name and not namespace_name :
+        result = list(client.list(resource_group_name, namespace_name))
+        return result
```

Comment:
What about no resource_group_name but namespace?
Is that handled somewhere else?

Reply:
added the check and raised the error.